app.rb

```diff
@@ -0,0 +1,71 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require './lib/peep'
+require './lib/user'
+require 'uri'
+require_relative './database_connection_setup'
+
+class Chitter < Sinatra::Base
```

Comment:
Your controller looks good.  Great to see you using RESTful route naming conventions, as well as keeping your routes concise and free of heavy logic